templates/iDRAC8/status.json

```
@@ -0,0 +1,8 @@
+{
+  "endpoint" : "/redfish/v1/Managers/iDRAC.Embedded.1",
```

iDRAC8 has the power state in another endpoint.
```suggestion
  "endpoint" : "/redfish/v1/Chassis/System.Embedded.1",
```